modules/languages/markdown/markdown.nix

```diff
@@ -40,6 +40,62 @@ in {
       };
     };
 
+    markdownPreview = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Enable markdown preview in neovim with markdown-preview.nvim";
```

Comment:
the enable option should use `mkEnableOption`, which handles bool type, false default and prefixes the description with "whether to enable"